egs/librispeech/ASR/pruned_transducer_stateless4/train.py

```diff
@@ -867,10 +867,6 @@ def run(rank, world_size, args):
         model = DDP(model, device_ids=[rank])
     model.device = device
 
-    if rank == 0:
-        model_avg.to(device)
-        model_avg.device = device
-
```

Comment:
Have you tested with this change?
I wonder whether you need to do `.to(torch.device('cpu'))` when updating this.

Reply:
Yes, I have tested it. `model_avg` is on cpu default, since it is a deepcopy of `model` before the line `model.to(device)`.

Reply:
OK.